lib/appoptics_apm/api/logging.rb

```diff
@@ -348,10 +350,12 @@ def log_event(layer, label, event, opts = {})
       def create_start_event(xtrace = nil)
         if AppOpticsAPM::XTrace.sampled?(xtrace)
           md = AppOpticsAPM::Metadata.fromString(xtrace)
+          md.padZeros
```

Comment:
TraceContext is not the same as Xtrace, TraceContext only reads and stores the incoming trace context and uses it for the tracing decision and maybe propagation
All the xtrace processing and formatting etc needs to be refactored, that part has npt been touched
I'm just padding with zeros so that it can be progated in traceparent